src/backend/src/prisma/schema.prisma

```diff
@@ -349,7 +349,7 @@ model Reimbursement_Request {
   dateCreated            DateTime                @default(now())
   dateDeleted            DateTime?
   dateOfExpense          DateTime
-  reimbursementsStatuses Reimbursement_Status[]
+  reimbursementStatuses  Reimbursement_Status[]
```

Comment:
the sql file says renamed without changes but this should probs cause a change right